src/medCore/process/medAbstractProcess.h

```diff
@@ -46,7 +46,10 @@ class MEDCORE_EXPORT medAbstractProcess : public dtkAbstractProcess
         NO_MESH,        //! Input can not be a mesh
         DATA_SIZE,      //! Inputs must be the same size
         MISMATCHED_DATA_TYPES, //! Inputs must be the same type
-        MISMATCHED_DATA_SIZES_ORIGIN_SPACING, //! Inputs should have the same sizes, origins, spacings
+        MISMATCHED_DATA_SIZES_ORIGIN_SPACING, //! Inputs should have the same size, origin, spacing
+        MISMATCHED_DATA_SIZE,    //! Inputs should have the same dimension
```

Comment:
Shouldn't the comment say "size" instead of "dimension" ?

Reply:
it's the same meaning here ^^

Reply:
No it's not. When we speak of the ***dimension*** of an image it refers to 2D, 3D etc. When we say ***dimensions**** we refer to the number of pixels in each dimension. If you want to keep the word "dimension" it should be plural, but I suggested "size" instead because that's the message sent to the user, and all the other comments mirror the message that is actually sent to the user except this one.